app/src/main/java/com/example/kevinschweitzer/myapplication/mvp/view/CalculatorView.java

```diff
@@ -0,0 +1,97 @@
+package com.example.kevinschweitzer.myapplication.mvp.view;
+
+import android.app.Activity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.example.kevinschweitzer.myapplication.R;
+import com.example.kevinschweitzer.myapplication.util.bus.RxBus;
+import com.example.kevinschweitzer.myapplication.util.bus.obervers.ClearObserver;
+import com.example.kevinschweitzer.myapplication.util.bus.obervers.EqualsObserver;
+import com.example.kevinschweitzer.myapplication.util.bus.obervers.NumberObserver;
+import com.example.kevinschweitzer.myapplication.util.bus.obervers.OperatorObserver;
+import com.example.kevinschweitzer.myapplication.util.operators.DivisionOperator;
+import com.example.kevinschweitzer.myapplication.util.operators.MinusOperator;
+import com.example.kevinschweitzer.myapplication.util.operators.MultiplicationOperator;
+import com.example.kevinschweitzer.myapplication.util.operators.PlusOperator;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+public class CalculatorView {
+
+    @BindView(R.id.text_operation) TextView operation;
+    @BindView(R.id.text_result) TextView result;
+    private Activity activity;
+
+    public CalculatorView(Activity activity){
+        this.activity = activity;
+        ButterKnife.bind(this, activity);
+    }
+
+    public void concatNumber(String str){
+        operation.setText(operation.getText() + str);
+    }
+
+    public void concatOperator(String str){
+        operation.setText(operation.getText() + " " + str + " ");
+    }
+
+    public void setResult(Integer result){
+        this.result.setText(result+"");
+    }
+
+    public void setResult(String error){
+        this.result.setText(error);
+    }
+
+    public void clear(){
+        operation.setText("");
+        result.setText("");
+    }
+
+    @OnClick({R.id.button_number0, R.id.button_number1, R.id.button_number2, R.id.button_number3, R.id.button_number4, R.id.button_number5,
```

Comment:
This is fine, but you have another manner to this do the same in a more simple manner. You could move this to the activity and call the presenter.
You dont need change, its only a comment